lib/ssl/test/ssl_cert_SUITE.erl

```diff
@@ -981,6 +984,25 @@ key_auth_ext_sign_only(Config) when is_list(Config) ->
     ssl_test_lib:basic_test(ClientOpts, ServerOpts, Config).
 
 %%--------------------------------------------------------------------
+cert_auth_in_first_ca() ->
+    [{doc,"Test cert auth will be available in firs ca in chain, make it happen by only having one"}].
```

Comment:
@IngelaAndin, thank you for fixing this for us. A minor comment about a typo in the test description for "first".

Reply:
Thanks for noticing, fixed.